src/pages/waiting-room/cat-b/__tests__/cat-b-waiting-room.spec.ts

```diff
@@ -3,44 +3,44 @@ import { ComponentFixture, async, TestBed, fakeAsync, tick } from '@angular/core
 import { IonicModule, NavController, NavParams, Config, Platform } from 'ionic-angular';
 import { NavControllerMock, NavParamsMock, ConfigMock, PlatformMock } from 'ionic-mocks';
 
-import { AppModule } from '../../../app/app.module';
-import { WaitingRoomPage } from '../waiting-room';
-import { AuthenticationProvider } from '../../../providers/authentication/authentication';
-import { AuthenticationProviderMock } from '../../../providers/authentication/__mocks__/authentication.mock';
+import { AppModule } from '../../../../app/app.module';
+import { CatBWaitingRoomPage } from '../cat-b-waiting-room';
```

Comment:
so with regard to my comment above this line would be something like 

import { WaitingRoomPageB } from '../waiting-room-b';

Reply:
I think I prefer `WaitingRoomPageB` over `CatBWaitingRoomPage`

Reply:
I'll switch it over , not a massive fan of the cat-b stuff.

Reply:
Do we not think self-explanatory and explicit is better than terse?

Reply:
> Based on the above comments have updated the folder structure to be
> 
> waiting-room
> -- category
> ---- b
> ----- waiting-room-b.xxx
> ---- be
> ------ waiting-room-be.xxx
> -- components
> -- waiting-room-actions.ts
> -- waiting-room.anlaytics.effects.ts
> -- waiting-room.module.ts

I liked the first approach better, imagine trying to update lots of these files at the same time. The only thing that distinguishes them is the file path! Could be problematic..

Reply:
Ammar has a good point. 
`WaitingRoomPageCatB`
`WaitingRoomPageCatBE`
`WaitingRoomPageCatC`
`WaitingRoomPageCatC1`
`WaitingRoomPageCatCE`
etc.